src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[],
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length === 0
+      ? (<b data-cy="NoCommentsMessage">No comments yet</b>)
+      : comments.map(comment => (
+        <CommentInfo key={comment.id} comment={comment} />
+      ))}
```

Comment:
You should always return <CommentInfo> component here and check this condition in Post info component